### DISCUSSION TITLE
Working with multiple databases - Examples? Tips? Best Practices?

### DISCUSSION THREAD

**Discussion by emallay** [simolus3/drift#1322] [Q&A] [answered]
Hi everyone. Could anyone point me towards some examples showing how to use moor with multiple open databases at once? 

What are best practices? Any tips or 'gotchas' I should be aware of?

Thanks in advance!

**Comment by simolus3** [comment, marked as answer]
Hi! Is there anything specifically you're interested in with multiple databases?

If you have different database classes pointing at different database files, there's nothing to worry about - you can just manage all of them independently. If you have the same database class opened multiple times (on different files), that's still fine. In that case, you should set `moorRuntimeOptions.dontWarnAboutMultipleDatabases = true`.
Really, the only thing you should avoid is opening the same logical database file multiple times. In all other cases, you can just open and close the database instances independently.

**Reply by emallay** [reply]
Thanks - that's good to know! I appreciate the quick feedback.
